resources/js/features/game-list/components/HubMainRoot/HubMainRoot.tsx

```diff
@@ -0,0 +1,78 @@
+import { dehydrate, HydrationBoundary } from '@tanstack/react-query';
+import type { FC } from 'react';
+
+import { usePageProps } from '@/common/hooks/usePageProps';
+
+import { useAutoUpdatingQueryParams } from '../../hooks/useAutoUpdatingQueryParams';
+import { useGameListState } from '../../hooks/useGameListState';
+import { usePreloadedTableDataQueryClient } from '../../hooks/usePreloadedTableDataQueryClient';
+import { hubGamesDefaultFilters } from '../../utils/hubGamesDefaultFilters';
+import { DataTablePaginationScrollTarget } from '../DataTablePaginationScrollTarget';
+import { HubGamesDataTable } from '../HubGamesDataTable';
+import { HubBreadcrumbs } from './HubBreadcrumbs';
+import { HubHeading } from './HubHeading';
+import { RelatedHubs } from './RelatedHubs';
+
+export const HubMainRoot: FC = () => {
+  const { auth, breadcrumbs, defaultDesktopPageSize, paginatedGameListEntries } =
+    usePageProps<App.Platform.Data.HubPageProps>();
+
+  const {
+    columnFilters,
+    columnVisibility,
+    pagination,
+    setColumnFilters,
+    setColumnVisibility,
+    setPagination,
+    setSorting,
+    sorting,
+  } = useGameListState(paginatedGameListEntries, {
+    alwaysShowPlayersTotal: true,
```

Comment:
Having this field visible adds an unnecessary horizontal scrollbar to my view:
![image](https://github.com/user-attachments/assets/4fe70ee6-5811-4c39-a1f4-50059176eef5)

Reply:
In latest, this field is no longer visible by default.